### DISCUSSION TITLE
When playing games, if i look too far to the right my mouse goes to my 2nd monitor..

### DISCUSSION THREAD

**Discussion by krees2006** [hyprwm/Hyprland#6644] [Q&A]
As the title says and if i were to click the 2nd monitor, my game goes crazy like the mouse wont work and keyboard kinda messes up. Hard to describe, but hopefully someone knows. 

Thank you to whoever sees this.

**Comment by fufexan** [comment]
Hyprland version? A similar bug was fixed yesterday on git.
